frontend/ui/widget/menu.lua

```diff
@@ -590,8 +590,12 @@ function Menu:_recalculateDimen()
     if self.dimen.h > Screen:getHeight() or self.dimen.h == nil then
         self.dimen.h = Screen:getHeight()
     end
+    self.inner_dimen = Geom:new{
+        w = self.dimen.w - 2 * self.border_size,
+        h = self.dimen.h - 2 * self.border_size,
+    }
```

Comment:
I assume _recalculateDimen is called when the screen size changes or on rotation ? so the need to fix up self.dimen and the new self.inner_dimen ?
And so also the need to have that same thing in CoverBrowser as the one in Menu:init() is not rock solid?

Reply:
That was my assumption too ;).

Reply:
Which might be wrong :) It's called again in `Menu:updateItems` - would need to see if this updating of items can change geometry. My new assumption would be that they shouldn't, and things like dimen shouldn't change. But ok, let it be :)
Menu:onScreenResize() calles Menu:init().

Reply:
On the other hand, if `dimen` does get updated, I don't see how we could go on using a stale `inner_dimen` anyway?

Reply:
Yes of course. So the question becomes: can self.dimen or Screen:getWidth/height change when this is called :)

Reply:
As in, does this (the whole dimen + inner_dimen thing) even belong in _recalcWhatverThisFunctionIsCalled in the first place?

Yeah, definitely a good question, especially if it's not related to rotation/resizes to all ;).